aws/resource_aws_lb_target_group.go

```diff
@@ -132,6 +132,7 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 						"path": {
 							Type:         schema.TypeString,
 							Optional:     true,
+							Default:      "/",
```

Comment:
This was intentionally removed in order to bring support for network load balancers where path is irrelevant.

See https://github.com/terraform-providers/terraform-provider-aws/commit/79467dc32944ceb42acaa60bb91da86fa974ef22#diff-375aea487c27a6ada86edfd817ba2401